src/Main.java

```diff
@@ -1,42 +1,54 @@
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Scanner;
+import java.util.Random;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code scrijven voor de hoofd-opdracht
+        HashMap<Integer, String> numericAlpha = new HashMap<>();
 
-        /* deze regel mag je weg halen voor de bonus opdracht. Onderstaande code is voor de bonus opdracht.
-        HashSet<Integer> secretnumber = randomnumbergenerator();
-        String stringnumber =  setToStringConverter(secretnumber);
-        System.out.println(stringnumber);
-        feedback();
-         deze regel mag je weg halen voor de bonus opdracht */
+        //Integer Array
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
 
-    }
+        //String Array
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
+
+        Translator translator = new Translator(numeric, alphabetic);
 
-    /*
-     Deze methode is voor de bonus opdracht.
-     */
-    public static void feedback(String stringnumber) {
+        boolean play = true;
+        String ongeldig = "Ongeldige invoer!";
         Scanner scanner = new Scanner(System.in);
-        StringBuilder feedback = new StringBuilder();
-        System.out.println("take a guess");
-        String guess = scanner.nextLine();
-        if (Objects.equals(guess, stringnumber)) {
-            System.out.println("gefeliciteerd je hebt het goed");
-        } else {
-            for (int i = 0; i < 4; i++) {
-                if (guess.substring(i, i + 1).equals(stringnumber.substring(i, i + 1))) {
-                    feedback.append("+");
-                } else if (stringnumber.contains(guess.substring(i, i + 1))) {
-                    feedback.append("0");
+
+        while (play) {
+            //Vraagt gebruiker om het programma te stoppen of het vertalen het starten
+            System.out.println("Type 'x' om te stoppen");
+            System.out.println("Type 'v' om te vertalen");
+            String input = scanner.nextLine();
+
+            if (Objects.equals(input, "x")) {
+                play = false;
+            } else if (Objects.equals(input, "v")) {
+                //Is altijd true, vraagt gebruiker om cijfer in te typen
+                System.out.println("Voer hier een cijfer tussen de 0 en 9 in.");
+                int inputGrade = scanner.nextInt();
+                scanner.nextLine();
+
+                //Eerste block wordt alleen uitgevoerd als de gebruiker een geldige cijfer heeft ingevoerd
+                //Tweede block wordt uitgevoerd als de gebruiker een ongeldige cijfer heeft ingevoerd
+                //Derde block wordt uitgevoerd als de gebruiker iets anders dan 'v' of 'x' invoert
+                if (inputGrade < 10) {
+                    String result = translator.Translate(inputGrade);
```

Comment:
Is translator vanuit hier te bereiken? Want deze staat nu in je feedback methode maar hij wordt gedeclareerd buiten je main methode als Class variabele toch?